fetch.bs

```diff
@@ -4950,12 +5049,23 @@ constructor must run these steps:
    to <var>method</var>.
   </ol>
 
+ <li><p>If <var>init</var>'s <code>signal</code> member is present, then set <var>signal</var> to
+ it.
```

Comment:
So, if you do `const r = new Request(otherRequest, { })` then otherRequest's signal will control aborting `r`. But if you do `const r = new Request(otherRequest, { signal })`, then `signal` will control aborting `r`, and `otherRequest` will have no impact at all. Is that correct? Is that what's tested?

Reply:
This does make the most sense to me. Note that we need this behavior pretty much like this due to `fetch()` invoking the `Request` constructor.